doc/modules/ensemble.rst

```diff
@@ -150,6 +150,46 @@ In contrast to the original publication [B2001]_, the scikit-learn
 implementation combines classifiers by averaging their probabilistic
 prediction, instead of letting each classifier vote for a single class.
 
+A competitive alternative to random forests are
+:ref:`histogram_based_gradient_boosting` (HGBT) models:
+
+-  Building trees: Random forests typically rely on deep trees (that overfit
+   individually) which uses much computational resources, as they require
+   several splittings and evaluations of candidate splits. Boosting models
+   build shallow trees (that underfit individually) which are faster to fit
+   and predict.
+
+-  Sequential boosting: In histogram-based gradient boosting (HGBT), the
+   decision trees are built sequentially, where each tree is trained to
+   correct the errors made by the previous trees. This is achieved by
+   assigning higher weights to the training instances that were misclassified
+   in the previous iteration, which helps the next tree to focus on those
+   instances and improve their classification. This process is repeated
+   iteratively until a predefined number of trees are built or until the
+   model's performance stops improving.
+
+   In contrast, random forests (RF) use an ensemble of decision trees, where
+   each tree is built independently and is not influenced by the other trees.
+   The final prediction is made by taking the majority vote of all the trees
+   in the ensemble. While RF can achieve high accuracy, they may require a
+   larger number of trees to achieve the same level of accuracy as HGBT. This
+   is because RF may not be able to correct errors made by previous trees as
+   effectively as HGBT, which can lead to a larger number of trees needed to
+   achieve the desired level of accuracy.
```

Comment:
I would say that explaining each method in depth is distracting, it is better to keep a short paragraph focusing on the differences and their consequences. Also notices that the abbreviation "HGBT" was already introduced above, so there is no need to redefine it.

```suggestion
-  Sequential boosting: In HGBT, the decision trees are built sequentially,
   where each tree is trained to correct the errors made by the previous ones.
   This allows them to iteratively improve the model's performance using
   relatively few trees. In contrast, random forests use a majority vote to
   predict the outcome, which can require a larger number of trees to achieve
   the same level of accuracy.
```